src/index.html

```diff
@@ -30,6 +30,11 @@
     });
   }
   </script>
+  
+  <script>
+  // Fixes undefined module function in SystemJS bundle
+  function module() {}
+  </script>
```

Comment:
Probably not the most elegant way, but I couldn't find the shims you mentioned and this basically fixed the only error thrown.